middleware/compress.go

```diff
@@ -382,6 +382,10 @@ func (cw *compressResponseWriter) Close() error {
 	return errors.New("chi/middleware: io.WriteCloser is unavailable on the writer")
 }
 
+func (cw *compressResponseWriter) Unwrap() http.ResponseWriter {
+	return cw.ResponseWriter
```

Comment:
This is the same pattern as in https://github.com/go-chi/chi/blob/77d709fcde559ca625d4e126478835a0e85d79a9/middleware/wrap_writer.go#L116.

Going to merge.